apps/condo/domains/common/components/GraphQlSearchInput.tsx

```diff
@@ -20,6 +20,7 @@ interface ISearchInputProps extends SelectProps {
     autoFocus?: boolean
     initialValue?: string
     formatLabel?: (option: { value: string, text: string }) => JSX.Element
+    maxTagCount?: 'responsive' | number
```

Comment:
SelectProps interface did not provide maxTagCount typedefs?

Reply:
Yes, antdesign type `SelectProps` contain that property, but `GraphQlSearchInput` component overrides all default props, so we have error with unknown prop type

Reply:
@Dimitreee ok, so I found solution to solve this. Please review again